docs/guides/03-quickstart.md

```diff
@@ -1,6 +1,8 @@
 ## Quickstart Guide
 
-Welcome to the CARTO.js documentation. CARTO.js is a JavaScript library that enables you to create custom location intelligence applications that leverage the power of the **[CARTO Engine](https://carto.com/pricing/engine/)** ecosystem.
+CARTO.js lets you create custom location intelligence applications that leverage the power of the **[CARTO Engine](https://carto.com/pricing/engine/)** ecosystem.
+
+This document details the CARTO.js v4. To update your v3 apps, please consult the [Upgrade Considerations]({{site.cartojs_docs}}/guides/upgrade-considerations/).
```

Comment:
remove "the":
"This document details CARTO.js v4."

Reply:
👍